WordPressPCL/Utility/HttpHelper.cs

```diff
@@ -14,7 +14,8 @@ namespace WordPressPCL.Utility
     /// </summary>
     public class HttpHelper
     {
-        private static readonly HttpClient _httpClient = new HttpClient();
+        private static readonly HttpClient _defaultHttpClient = new HttpClient();
```

Comment:
Technically you are right, and that would be better solution, but for existing library users requirement to pass HttpClient to constructor would be a breaking change. Let's say somebody create multiple WordPressClient instances and relies on the fact that just single HttpClient process them. So that's why I keep that code as it is to maintain existing behaviour.

Reply:
no I agree that the existing constructors shouldn't change.
you're right, that way existing behavior is maintained.